django/http/multipartparser.py

```diff
@@ -145,6 +147,11 @@ def parse(self):
         old_field_name = None
         counters = [0] * len(handlers)
 
+        # To count the number of fields
+        num_inserted_fields = 0
```

Comment:
I'm not sure what "inserted" references in "num_inserted_fields". Something like "num_post_keys". I think we should use "keys" in most place we talk about "fields" and modify any docs which suggest this is specific to Django forms rather than POST requests in general.